testForm.html

```diff
@@ -0,0 +1,715 @@
+<link rel="stylesheet" href="styles.css">
```

Comment:
I know it's a test form but we need the <html> and <head> and doctype and <body> standard headers

Reply:
body tag and html tag and head tag. Comments remove tags if I put them in